### DISCUSSION TITLE
Update of samples

### DISCUSSION THREAD

**Discussion by Cloud33** [databendlabs/openraft#697] [General]
I'm a beginner, so I started [raft-kv-memstore](https://github.com/datafuselabs/openraft/blob/main/examples/raft-kv-memstore/README.md) To learn openraft, I restored all the code and found that I could not run it from openraft 0.7.4. It has many errors, probably because the example uses the old version of openraft.

If you can, please update the example of openraft to the latest version

thank you

**Comment by drmingdrmer** [comment]
What's the openraft version in your `Cargo.lock` ?

Openraft 0.8 changed a lot of its API. Make sure you copied from the 0.7 raft-kv-memstore and compiled it with openraft 0.7, or copied from 0.8 raft-kv-memstore and compiled with openraft 0.8;

**Reply by Cloud33** [reply]
use openraft 0.7.4, Do you mean that I should not copy code from Main, but from openraft 0.7.4 branch?

**Reply by drmingdrmer** [reply]
Yes. memstore version and openraft version must match.

I'd suggest using the latest release v0.8, which was published days before.
Meanwhile branch `main` points to release-0.8;

**Reply by Cloud33** [reply]
Thank you, great project
